### ISSUE TITLE
verifyEmpiricalToTheoretical fails if the input sequence does not contain all possible states from the markovchain object

### ISSUE THREAD

**Issue by ebbertd** [spedygiorgio/markovchain#191]
Given the following example:
```
library(markovchain)
mc <- matrix(c(
  1 / 10, 7 / 10, 1 / 10, 1 / 10,
  1 / 10, 1 / 10, 4 / 10, 4 / 10,
  1 / 10, 5 / 10, 1 / 10, 3 / 10,
  1 / 10, 5 / 10, 3 / 10, 1 / 10
),
byrow = TRUE,
nrow = 4
)
rownames(mc) <- c(1:4)
colnames(mc) <- c(1:4)
theoreticalMc <- as(mc, "markovchain")
sequence <- c(1, 1, 1, 1, 1, 1, 2, 1, 1, 1, 1, 1, 1, 4, 1, 1, 1, 1, 1, 1, 1)
verifyEmpiricalToTheoretical(data = sequence, object = theoreticalMc)
```
The following error is produced:
```
Fehler in data[match(rownames(data), names(object)), ] : 
  Indizierung außerhalb der Grenzen
```
This is due to the missing state in the sequence that is present in the markovchain object. The created matrix for the input should thus be adjusted to include all possible states from the markovchain object.

**Comment by spedygiorgio** [comment]
thanks for fixing it @ebbertd , may you also write some unit tests for it?

**Comment by ebbertd** [comment]
See pull request #198

**Comment by spedygiorgio** [comment]
This issue has been addressed by PR #235, which was merged on 2026-08-15.

`verifyEmpiricalToTheoretical()` now aligns empirical transition counts with the full state space of the theoretical Markov chain, including states that are absent from the observed sequence. Regression tests cover missing-state inputs.

Closing as fixed by #235.
